integration-tests/batch-api.spec.ts

```diff
@@ -12,7 +12,7 @@ CONFIGS().forEach(({ lib, rpc, setup, knownBaker, createAddress }) => {
       done()
     })
     it('Verify simple batch transfers with origination', async (done) => {
-      const batch = await Tezos.batch()
+      const batch = await Tezos.contract.batch()
```

Comment:
We already have a file to test the contract.batch: https://github.com/ecadlabs/taquito/blob/master/integration-tests/contract-batch.spec.ts
This one should not be modified. We will delete it when we remove the deprecated method.

Reply:
Gotcha, it's been rolled back